test/wallet-http-test.js

```diff
@@ -71,11 +72,17 @@ for (const witnessOpt of witnessOptions) {
 
     it('should create wallet', async () => {
       const info = await wclient.createWallet(name, {witness: witnessOpt});
+      wallets.push(name);
       assert.strictEqual(info.id, name);
       wallet = wclient.wallet(name, info.token);
       await wallet.open();
     });
 
+    it('should list wallets', async () => {
+      const info = await wclient.getWallets();
+      assert.equal(JSON.stringify(info), JSON.stringify(wallets));
```

Comment:
I believe `assert.deepStrictEqual()` will test that two arrays are identical.

Reply:
yes, that works! Also, for some reason, the implementation of assert with and without 'strict' is the exact same thing